docker/mysql/bugzilla.cnf

```diff
@@ -1,8 +1,9 @@
 [mysqld]
 max_allowed_packet = 1G
-innodb_file_per_table = 1
-innodb_large_prefix   = 1
```

Comment:
additional system variables that are deprecated/removed can be prefixed with `loose-` where the server, mysql or mariadb, will not error  if the variable has an invalid value or is not there any more.

So
```
loose_innodb_large_prefix = ON`
```

Reply:
This is good to know! The older versions understand that and set it without the loose in front?
Also: underscore or hyphen? Your two examples are different.

Reply:
Yes, loose was there for a long time (at least 5.5 - just checked). In the config file and command line, underscores and hyphens are identically handled.